python/ray/experimental/raysort/main.py

```diff
@@ -30,25 +34,43 @@ def get_args():
     )
     parser.add_argument(
         "--total_data_size",
-        default=1_000_000_000,
+        default=512 * 1000 * 1024 * 1024,
         type=ByteCount,
         help="partition size in bytes",
     )
     parser.add_argument(
         "--num_mappers",
-        default=4,
+        default=128,
         type=int,
         help="number of map tasks",
     )
     parser.add_argument(
         "--num_reducers",
+        default=16,
+        type=int,
+        help="number of reduce actors",
+    )
+    parser.add_argument(
```

Comment:
I think it would be more intuitive to say this as maximum number of rounds in flight.

Ideally, I think we always want this to be 2, so that we are producing map tasks for the next round while shuffling the current round. (and setting M, R, and N appropriately to saturate memory).

Reply:
Let's try benchmarking with num_concurrent_rounds=1 and 2.

So 1 should be very conservative and make sure that all of the previous map results have been evicted before the next round is submitted, and we should expect to see no spilling at all.

With 2, we might see spilling if the next round's map tasks are scheduled too soon, but it would be good to measure the difference with this benchmark.